<launch>
  <arg name="robot_namespace"/>
  <arg name="rbtx" default="$(optenv GM_PARAM_RBT_X 1)"/>
  <arg name="rbty" default="$(optenv GM_PARAM_RBT_Y 2)"/>

  <node pkg="stdr_robot" type="robot_handler" name="$(anon robot_spawn)" args="add $(find dynamic_gap)/stdr_robots/robots/agent.xml $(arg rbtx) $(arg rbty) 0.0" />

  <!-- 
  <node pkg="dynamic_gap" type="follow_global_path.py" name="$(anon follow_global_path)" output="screen">
	<param name="robot_namespace" value="$(arg robot_namespace)"/>
  </node>
  -->

</launch>
